backend/fms_core/viewsets/sample_pooled.py

```diff
@@ -0,0 +1,117 @@
+from rest_framework import viewsets, serializers
+from rest_framework.decorators import action
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.pagination import LimitOffsetPagination
+from rest_framework.exceptions import APIException
+from fms_core.models import Biosample, DerivedSample, DerivedBySample, Index, Library, LibraryType, Project, Sample, SampleKind
+from fms_core.filters import PooledSampleFilter
+
+# Custom serializers
+# These serializers serialize the data we want to receive in pool samples.
+
+EXCLUDE_BOOKKEEPING = ['created_at', 'updated_at', 'created_by', 'updated_by', 'deleted']
```

Comment:
don't exclude those fields from serializers unless you do a serializer that is meant solely for the end user (model export for users)